xapian-applications/omega/omindexcheck.cc

```diff
@@ -196,8 +257,18 @@ main(int argc, char** argv)
 	Xapian::Document doc = db.get_document(did);
 	auto iter = tests.find(url);
 	if (iter != tests.end()) {
-	    succeed &= compare_test(iter->second, doc, url);
+	    test_result outcome = compare_test(iter->second, doc, url);
```

Comment:
I'd probably call this `individual_result` or something. It's not going to be clear to someone just reading the variable names what the difference between "outcome" and "result" is.

Reply:
Thank you for pointing this out. I have changed the variable name to `individual_result` instead of `outcome`.